### PR TITLE
Code coverage build config file

### PR DESCRIPTION
**Pull request by mazimkhan**

### CODE REVIEW

File: coverage.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "debug":{
+        "options" : {
+            "coverage" : {
+                "modules" : {
+                    "mbed-drivers" : true
+                }
+            }
+        }
+    },
+    "mbed-os": {
```
Comment (0xc0170) [inline coverage.json]: This config is aligned differently than the above? Is yotta default alignment 2 spaces?

Comment (PrzemekWirkus) [inline coverage.json]: This is ```json``` file, who cares how many spaces are there in it...

**Comment by PrzemekWirkus** [comment]
+1

@mazimkhan You can squash those two commits to one. @bogdanm  probably would like to have clean history.

File: coverage.json (hunk 1 above)
Comment (0xc0170) [inline coverage.json]: It improves readability, same applies to our "code" files, we should care :facepunch:

**Pull request merged**